src/System.Private.Xml/src/System/Xml/Dom/XPathNodeList.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It could be, my code optionally re-writes the files to make the lines commented out (will change to deleted out during the week). I did copy some example C# code from a Microsoft help page on how to detect encoding but couldn't see anything about the BOM and wondered at the time what would happen. Suppose it could also be the line ending but obviously it happens in only one place. Time for a binary diff.